### ISSUE TITLE
Protractor tests don't actually run but report passing (webpack branch)

### ISSUE THREAD

**Issue by caseyhoward** [angular/angular-cli#2072]
1. Linux Mint 17.1

2.  angular-cli: 1.0.0-beta.11-webpack.8
  node: 6.5.0
  os: linux x64

3. Ran "npm init" then "ng init" in a new directory.
Added the following test to e2e/app.e2e-spec.ts:
  ```ts
  it('should blow up', () => {
    page.navigateTo();
    expect(page.getParagraphText()).toEqual('app doesn\'t work!');
  });
  ```
Ran "npm start" in one tab and "npm e2e" in another

4. 
The error is that there is no error. The test I added should have failed.
```
$ npm run e2e

> blah@0.0.0 pree2e /home/.../blah
> webdriver-manager update

[normal looking chrome driver output]

> blah@0.0.0 e2e /home/...
> protractor

[12:16:55] I/direct - Using ChromeDriver directly...
[12:16:55] I/launcher - Running 1 instances of WebDriver
Spec started

  blah App
    ✓ should display message saying app works
    ✓ should blow up

Executed 2 of 2 specs SUCCESS in 0.025 sec.
[12:16:56] I/launcher - 0 instance(s) of WebDriver still running
[12:16:56] I/launcher - chrome #01 passed
```

5. It appears that it isn't doing anything at all. No matter how many steps you add to your test in just exits immediately.

**Comment by kv113813** [comment]
I do have similar issue with Ubuntu + selenium, protractor setup with headless chrome. It passes all the test cases where many shows up failed in other windows/MAC setups and also passes in fact intensionally made failed test cases. 
<img width="489" alt="screen shot 2016-09-12 at 4 57 27 pm" src="https://cloud.githubusercontent.com/assets/13733586/18452710/1392f3f0-790a-11e6-830d-35431ddb6e76.png">

**Comment by christiandreher** [comment]
Fixed with protractor 4.0.5, make an update or revert to 3

**Comment by filipesilva** [comment]
Fixed in https://github.com/angular/angular-cli/pull/2054
